panels/site_sections/schedule.py

```diff
@@ -173,6 +173,21 @@ def form(self, session, message='', panelists=(), **params):
         event = session.event(params, allowed=['location', 'start_time'])
         if 'name' in params:
             session.add(event)
+
+            # Associate a panel app with this event, and if the event is new, use the panel app's name and title
+            if 'panel_id' in params:
+                add_panel = session.panel_application(id=params['panel_id'])
+                add_panel.event_id = event.id
+                session.add(add_panel)
+                if event.is_new:
+                    event.name = add_panel.name
+                    event.description = add_panel.description
+                    for pa in add_panel.applicants:
+                        if pa.attendee_id:
```

Comment:
For practical reasons, panelists without associated attendees cannot be associated with an event.